hack/run_test

```diff
@@ -259,7 +263,7 @@ function run_tests() {
 function run_slave() {
   local master_ip="$1"; shift
   local suffix="$1"; shift
-  docker run $cluster_args -e POSTGRESQL_MASTER_IP=${master_ip} \
+  docker run $cluster_args -p 5432 -e POSTGRESQL_MASTER_IP=${master_ip} \
```

Comment:
Thanks for spotting this. This is actually leftover code from me trying to figure out why the the new slave doesn't connect to the master. 
Will remove the change in the next push as it is not needed.